charts/fluid-dataprocess/common/templates/job.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ printf "%s-job" .Release.Name }}
+  labels:
+    release: {{ .Release.Name }}
+    role: dataprocess-job
+    app: fluid-dataprocess
+    targetDataset: {{ required "targetDataset should be set" .Values.dataProcess.targetDataset }}
+spec:
+  backoffLimit: 3
+  completions: 1
+  parallelism: 1
+  template:
+    {{- if .Values.dataProcess.jobProcessor.podTemplate }}
+{{- toYaml .Values.dataProcess.jobProcessor.podTemplate | nindent 4 }}
+    {{- else if .Values.dataProcess.scriptProcessor }}
+    metadata:
```

Comment:
Should these metadata still be set when using jobProcessor?

Reply:
You're right! Nice suggestion, Thx! Now using PodSpec instead.